rust/benches/array_from_vec.rs

```diff
@@ -17,15 +17,14 @@
 
 #[macro_use]
 extern crate criterion;
-
 use criterion::Criterion;
 
 extern crate arrow;
 
-use arrow::array::*;
-use arrow::array_data::ArrayDataBuilder;
-use arrow::buffer::Buffer;
-use arrow::datatypes::*;
+use arrow::arrow::array::*;
```

Comment:
@sunchao I wonder how could We avoid having this redundant `arrow::` namespace? 
What other structuring approaches do We have (e.g. a cargo workspace)? What are the advantages/disadvantages of them?